test/unit/definitions/contested/CaseEventToFields.test.js

```diff
@@ -4,7 +4,7 @@ const { find } = require('lodash');
 
 const caseEvent = Object.assign(require('definitions/contested/json/CaseEvent/CaseEvent.json'), []);
 const caseEventProd = Object.assign(require('definitions/contested/json/CaseEvent/CaseEvent-prod.json'), []);
-const caseEventNonProd = Object.assign(require('definitions/contested/json/CaseEvent/CaseEvent-nonprod.json'), []);
+const caseEventNonProd = Object.assign(require('definitions/contested/json/CaseEvent/CaseEvent-OfflineNotifications-non-prod.json'), []);
```

Comment:
rename nonprod file or this will show in prod

Reply:
Super spot - @HarryH96 has lost his crown as the 'CCD Expert'. Changes made 👍